apps/condo/domains/ticket/hooks/useTicketExportToPdfTask.tsx

```diff
@@ -269,9 +280,22 @@ export const useTicketExportToPdfTask: UseTicketExportToPdfTaskType = (props)  =
     }, [])
 
     const handleSaveToPdfTask = useCallback(() => {
+        const selectedTicketsCount = ticketId ? 1 : get(where, ['id_in', 'length'], null)
+        const eventProperties = {
+            selectedCommentCount: haveAllComments ? 'all' : checkedCommentIds.length < 1 ? 'nothing' : 'some',
```

Comment:
nested ternary are hard to read and understand. at least you could use brackets to make it more readable